patterns/gitleaks/7.6.1/98-general.toml

```
@@ -98,6 +98,18 @@
   regex = '''[a-zA-Z0-9\/\+]*(YXdzX2FjY2Vzc19rZXlfaW|F3c19hY2Nlc3Nfa2V5X2lk|hd3NfYWNjZXNzX2tleV9pZ)[a-zA-Z0-9\/\+]*(QUtJQ|FLSU|BS0lB|UVV0Sl|FVdEpR|RVXRKU|RkxTV|ZMU1|GTFNV|QlMwbE|JTMGxC|CUzBsQ)[a-zA-Z0-9\/\+]*(YXdzX3NlY3JldF9hY2Nlc3Nfa2V5|F3c19zZWNyZXRfYWNjZXNzX2tle|hd3Nfc2VjcmV0X2FjY2Vzc19rZX)[a-zA-Z0-9\/\+]+={0,2}'''
   tags = ["alert:repo-owner", "key", "aws", "type:secret"]
 
+[[rules]]
+  # find base64 encoded versions of OpenSSH private keys
+  description = 'Base64 Encoded OpenSSH Private Key'
```

nit: since a lot of the other rules tell you what the b64 shifted portions are

```suggestion
  # Base64 shifted versions of "BEGIN OPENSSH PRIV" and "END OPENSSH PRIV"
  description = 'Base64 Encoded OpenSSH Private Key'
```

---

I like it. Adding the change